### DISCUSSION TITLE
카카오 로그인 redirect URI 관련 질문

### DISCUSSION THREAD

**Discussion by bluegown** [100-hours-a-week/discussions#59] [개발] [answered]
현재 저희가 짠 로직입니다.

1. 프론트엔드에서 카카오에 요청을 해서 인가 코드를 받아옴
2. 인가 코드를 백엔드에게 넘김
3. 백엔드에서는 그 인가 코드를 통해 카카오에서 토큰을 발급받음
4. 카카오에서 받은 accessToken을 통해 유저 정보를 받아옴
5. 이후 프론트엔드에게 JWT를 발급해서 보내줌

-> 백엔드 서버 , 프론트엔드 로컬 / 백엔드 로컬 , 프론트엔드 로컬까지는 모두 수월하게 진행됩니다.

근데 서버 - 서버로 가면 창이 이동되지가 않고 JWT 토큰에 대한 정보가 화면에 다 나타나고 , 다른 창으로 이동이 안됩니다

해결책을 잘 모르겠습니다ㅠㅠ

**Comment by ej31** [comment]
- 서버 -> 서버로가면 창이 이동되지 않는다는게 이해가 되지 않습니다.
- 만약 설명하기 어려우시다면 영상이라도 찍어서 올려주십쇼 하하

**Reply by ej31** [reply, marked as answer]
# 박수치면서 좋아하시는걸 보니 해결하신 것 같군요.. 🤡
